spec/slimdown/page_spec.rb

```diff
@@ -58,6 +58,21 @@
     end
   end
 
+  describe '#parent' do
+    it 'returns parent' do
+      page = Slimdown::Page.find('test/child')
+      parent = page.parent
+      expect(parent).to be_a(Slimdown::Page)
```

Comment:
If reasonable, could you check that the parent's slug is `'test'`? Probably `expect(parent.path).to eq 'test'`.